src/pages/returnDirection.tsx

```diff
@@ -97,6 +97,14 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
         throw new Error(`No service info could be retrieved for nocCode: ${nocCode} and lineName: ${lineName}`);
     }
 
+    // Remove journeys with duplicate start and end points for display purposes
+    service.journeyPatterns = service.journeyPatterns.filter(
```

Comment:
Moving this to do it before the length check, otherwise if the only journey patterns are duplicates it won't redirect you